src/lava/proc/sdn/process.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import typing
+from enum import IntEnum, unique
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.variable import Var
+from lava.magma.core.process.ports.ports import InPort, OutPort
+
+
+@unique
+class ACTIVATION_MODE(IntEnum):
+    """Enum for synapse sigma delta activation mode. Options are {``ReLU : 0``}.
```

Comment:
1. ReLU is 1. So the dict in the docstring should say {``ReLU : 1``}
2. Are we planning to extend this to more activations? If yes, maintaining an Enum will get unwieldy beyond 4 or 5 neuron models.
3. Also, "ACTIVATION_MODE" sort of makes SigmaDelta as the "neuron model" with just 2 activation modes. If we want to apply Sigma and Delta around an arbitrary neuron model, we can all this Enum something else.

Reply:
That's correct. fixed ReLU enum. We have individual Sigma and Delta processes as well which can be wrapped around any arbitrary process. However, for eventual implementation in ucode, we would need separate implementation for each of them. So the thinking is we can use the enum to switch between different ucode implementations that we support and extend the enums accordingly.